opennms-model/src/main/java/org/opennms/netmgt/model/events/EventUtils.java

```diff
@@ -416,6 +416,22 @@ public static Event createNodeUpdatedEvent(String source, Integer nodeId, String
         }
         return bldr.getEvent();
     }
+    
+    /**
+     * <p>createRescanNodeEvent</p>
+     *
+     * @param source a {@link java.lang.String} object.
+     * @param nodeId a {@link java.lang.Integer} object.
+     * @return a {@link org.opennms.netmgt.xml.event.Event} object.
+     */
```

Comment:
Please don't add these auto-genereated comments.

Reply:
I removed this one, the others I left to keep the style consistent within classes. I can remove the others too.